src/sentry/snuba/metrics/fields/base.py

```diff
@@ -478,7 +476,8 @@ class DerivedMetricExpression(DerivedMetricExpressionDefinition, MetricExpressio
     def _raise_entity_validation_exception(self, func_name: str) -> None:
         raise DerivedMetricParseException(
             f"Method `{func_name}` can only be called on instance of "
-            f"{self.__class__.__name__} {self.metric_name} with a `projects` attribute."
+            f"{self.__class__.__name__} "
+            f"{get_reverse_mri(self.metric_name)} with a `projects` attribute."
```

Comment:
Should we rename `self.metric_name` to `self.metric_mri`?

Reply:
Will also do in a follow up PR